src/Tasks/Common/Resources/Strings.resx

```diff
@@ -346,4 +346,10 @@
 {1} - Restored version of platform package
 {2} - Current version of platform package</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="PackAsToolOnlySupportNetcoreapp" xml:space="preserve">
```

Comment:
Pending string review

Reply:
@KathleenDollard for this 2 strings
"DotnetTool only supports .NET Core."
"DotnetTool does not support target framework lower than netcoreapp2.1."

Reply:
Looks good

Reply:
What is "DotnetTool" in this message?  Is this supposed to refer to the `PackAsTool` property?

Reply:
Global tools feature officially is called DotnetTool, since it is the nuget package type. And Rob once asked in mail for official name, that is the one. However, it does not stick that well